ArduPlane/control_modes.cpp

```diff
@@ -220,3 +222,24 @@ bool Plane::fly_inverted(void)
     }
     return false;
 }
+
+/*
+  does current control mode allow autotuning via switch?
+  */
+bool Plane::mode_allows_autotuning(void)
```

Comment:
better to use a method on the mode, like does_auto_throttle(), default false and make it true for the ones you want to allow

Reply:
done